src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+  }
+
+  &__info {
+    margin-top: 40px;
+
+    &-name {
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+    }
+
+    &-product-code {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: $secondary-text-color;
+      margin-top: 4px;
+      text-align: left;
```

Comment:
The `text-align: left;` declaration is not necessary here if the default text alignment is not changed elsewhere. It's better to avoid setting properties to their default values unless you are overriding a previous setting.